src/modules/commander/commander_params.c

```diff
@@ -243,9 +243,11 @@ PARAM_DEFINE_FLOAT(COM_DISARM_LAND, 2.0f);
 PARAM_DEFINE_FLOAT(COM_DISARM_PRFLT, 10.0f);
 
 /**
- * GPS preflight check
+ * Arming without GNSS configuration
```

Comment:
This gives people a fighting chance of knowing what it is from the short description. If this was just enable or disable I'd probably call it circuit breaker.